### PR TITLE
Cluster estimation new object types

### PR DESCRIPTION
**Pull request by Aleksa-M**
Added string field in ObjectInWorld class to denote which kind of object in the real world a particular instance is. Also changed ClusterEstimation.run method to assign the object type to each instance of ObjectInWorld it creates (it assigns each ObjectInWorld a value of "object".)

### CODE REVIEW

File: modules/object_in_world.py [hunk 1]
```diff
@@ -38,3 +42,4 @@ def __init__(
         self.location_x = location_x
         self.location_y = location_y
         self.spherical_variance = spherical_variance
+        self.object_type = object_type
```
Comment (Xierumeng) [inline modules/object_in_world.py:45]: Make this `label` and of type integer following convention (e.g. `detections_and_time` , `detection_in_world` ).

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

File: modules/object_in_world.py [hunk 2]
```diff
@@ -12,23 +12,27 @@ class ObjectInWorld:
 
     @classmethod
     def create(
-        cls, location_x: float, location_y: float, spherical_variance: float
+        cls, location_x: float, location_y: float, spherical_variance: float, label=0
```
Comment (Xierumeng) [inline modules/object_in_world.py:15]: Remove default value and add type annotation (Flake8 will not pass).

File: modules/object_in_world.py [hunk 3]
```diff
@@ -12,23 +12,27 @@ class ObjectInWorld:
...
     ) -> "tuple[bool, ObjectInWorld | None]":
         """
         location_x, location_y: Location of the object.
         spherical_variance: Uncertainty of the location.
+        label: type of object in real world.
```
Comment (Xierumeng) [inline modules/object_in_world.py:20]: Capitalize: `Type`

File: modules/object_in_world.py [hunk 4]
```diff
@@ -12,23 +12,27 @@ class ObjectInWorld:
...
+            cls.__create_key, location_x, location_y, spherical_variance, label
+        )
 
     def __init__(
         self,
         class_private_create_key: object,
         location_x: float,
         location_y: float,
         spherical_variance: float,
+        label=0,
```
Comment (Xierumeng) [inline modules/object_in_world.py:35]: Remove default value and add type annotation.

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

**Comment by Aleksa-M** [comment]
new PR made

**Pull request closed without merging**